clustered/build.gradle

```diff
@@ -15,14 +15,13 @@
  */
 
 ext {
-    terracottaPlatformVersion = '5.0.0.beta'
-    terracottaApisVersion = '1.0.0.beta'
+    terracottaPlatformVersion = '5.0.3.beta'
+    terracottaApisVersion = '1.0.3.beta'
     terracottaCoreVersion = '5.0.0-beta2'
-    coordinatorVersion = terracottaPlatformVersion
     clusteredMapVersion = terracottaPlatformVersion
     offheapResourceVersion = terracottaPlatformVersion
     entityApiVersion = terracottaApisVersion
-    entityTestLibVersion = terracottaApisVersion
+    entityTestLibVersion = '1.0.3.beta'
```

Comment:
@chrisdennis are you talking about the versioning ? 
.z means api changes, while .beta, .beta2, etc. means fixes releases
all tc components are .0.3.beta currently